main/main.go

```diff
@@ -1,11 +1,10 @@
 package main
 
 import (
-	"context"
+	"github.com/braintree/manners"
```

Comment:
https://github.com/braintree/manners#known-issues, I guess we not need to worry about this issue rite at least for now?

Reply:
right, I think we are fine now. Manners is known for its support of graceful shut down.